### ISSUE TITLE
[PATCH] Fix typos (spelling errors) in ext/*.

### ISSUE THREAD

**Issue by p5pRT** [Perl/perl5#10985]
Migrated from [rt.perl.org#81882](https://rt-archive.perl.org/perl5/Ticket/Display.html?id=81882) (status was 'resolved')

Searchable as RT81882$

**Comment by p5pRT** [comment]
### From @pjacklam



---
 ext/B/B\.xs                                         |    2 \+\-
 ext/B/B/Concise\.pm                                 |    2 \+\-
 ext/B/t/OptreeCheck\.pm                             |    4 \+\+\-\-
 ext/B/t/concise\-xs\.t                               |    4 \+\+\-\-
 ext/B/t/optree\_concise\.t                           |    2 \+\-
 ext/B/t/optree\_varinit\.t                           |    2 \+\-
 ext/Devel\-Peek/Changes                             |    2 \+\-
 ext/Devel\-Peek/t/Peek\.t                            |    6 \+\+\+\-\-\-
 ext/DynaLoader/dl\_aix\.xs                           |    2 \+\-
 ext/DynaLoader/dl\_dlopen\.xs                        |    2 \+\-
 ext/DynaLoader/dl\_vmesa\.xs                         |    2 \+\-
 ext/File\-Glob/bsd\_glob\.c                           |    2 \+\-
 ext/Hash\-Util\-FieldHash/lib/Hash/Util/FieldHash\.pm |    2 \+\-
 ext/Hash\-Util\-FieldHash/t/11\_hashassign\.t          |    2 \+\-
 ext/IPC\-Open3/lib/IPC/Open3\.pm                     |    2 \+\-
 ext/IPC\-Open3/t/IPC\-Open3\.t                        |    2 \+\-
 ext/Opcode/Opcode\.xs                               |    2 \+\-
 ext/POSIX/POSIX\.xs                                 |    2 \+\-
 ext/POSIX/lib/POSIX\.pm                             |    2 \+\-
 ext/POSIX/t/math\.t                                 |    2 \+\-
 ext/POSIX/t/time\.t                                 |    2 \+\-
 ext/SDBM\_File/sdbm/biblio                          |    2 \+\-
 ext/SDBM\_File/sdbm/dbe\.c                           |    2 \+\-
 ext/SDBM\_File/sdbm/sdbm\.3                          |    2 \+\-
 ext/SDBM\_File/sdbm/sdbm\.c                          |    4 \+\+\-\-
 ext/Socket/Socket\.pm                               |    2 \+\-
 ext/XS\-APItest/APItest\.xs                          |    2 \+\-
 ext/XS\-APItest/exception\.c                         |    4 \+\+\-\-
 ext/XS\-APItest/t/my\_cxt\.t                          |    2 \+\-
 ext/mro/mro\.xs                                     |    2 \+\-
 ext/re/t/regop\.t                                   |    2 \+\-
 31 files changed\, 37 insertions\(\+\)\, 37 deletions\(\-\)

<details><summary>Inline Patch</summary>

```diff
diff --git a/ext/B/B.xs b/ext/B/B.xs
index c5d971b..c1071c5 100644
--- a/ext/B/B.xs
+++ b/ext/B/B.xs
@@ -1641,7 +1641,7 @@ PV(sv)
 	       to avoid tripping up over variable names in the pads.  */
 	    if((SvLEN(sv) && len >= SvLEN(sv))) {
 		/* It claims to be longer than the space allocated for it -
-		   presuambly it's a variable name in the pad  */
+		   presumably it's a variable name in the pad  */
 		len = strlen(p);
 	    }
 #endif
diff --git a/ext/B/B/Concise.pm b/ext/B/B/Concise.pm
index 6eae872..3849a18 100644
--- a/ext/B/B/Concise.pm
+++ b/ext/B/B/Concise.pm
@@ -306,7 +306,7 @@ sub compileOpts {
 		if (!$Config::Config{usedl}
 		    && keys %{$pkg.'::'} == 1
 		    && $pkg->can('bootstrap')) {
-		    # It is something that we're staticly linked to, but hasn't
+		    # It is something that we're statically linked to, but hasn't
 		    # yet been used.
 		    eval "require $pkg";
 		}
diff --git a/ext/B/t/OptreeCheck.pm b/ext/B/t/OptreeCheck.pm
index a85c5fc..bae9cb6 100644
--- a/ext/B/t/OptreeCheck.pm
+++ b/ext/B/t/OptreeCheck.pm
@@ -317,7 +317,7 @@ our %gOpts = 	# values are replaced at runtime !!
      selftest	=> 'self-tests mkCheckRex vs the reference rendering',
 
      fail	=> 'force all test to fail, print to stdout',
-     dump	=> 'dump cmdline arg prcessing',
+     dump	=> 'dump cmdline arg processing',
      noanchors	=> 'dont anchor match rex',
 
      # array values are one-of selections, with 1st value as default
@@ -679,7 +679,7 @@ sub mkCheckRex {
     # make targ args wild
     $str =~ s/\[t\d+\]/[t\\d+]/msg;
 
-    # escape bracing, etc.. manual \Q (doesnt escape '+')
+    # escape bracing, etc.. manual \Q (doesn't escape '+')
     $str =~ s/([\[\]()*.\$\@\#\|{}])/\\$1/msg;
     # $str =~ s/(?<!\\)([\[\]\(\)*.\$\@\#\|{}])/\\$1/msg;
 
diff --git a/ext/B/t/concise-xs.t b/ext/B/t/concise-xs.t
index 9b42325..fa120b0 100644
--- a/ext/B/t/concise-xs.t
+++ b/ext/B/t/concise-xs.t
@@ -20,7 +20,7 @@ and the stash is scanned for the function-names in that package.
 Each value in %$testpkgs is a hash-of-lists (HoL) whose keys are
 implementation-types and values are lists of function-names of that type.
 
-To keep these HoLs smaller and more managable, they may carry an
+To keep these HoLs smaller and more manageable, they may carry an
 additional 'dflt' => $impl_Type, which means that unnamed functions
 are expected to be of that default implementation type.  Those unnamed
 functions are known from the scan of the package stash.
@@ -38,7 +38,7 @@ If a function is implemented differently on different platforms, the
 test for that function will fail on one of those platforms.  These
 specific functions can be skipped by a 'skip' => [ @list ] to the HoL
 mentioned previously.  See usage for skip in B's HoL, which avoids
-testing a function which doesnt exist on non-threaded builds.
+testing a function which doesn't exist on non-threaded builds.
 
 =head1 OPTIONS AND ARGUMENTS
 
diff --git a/ext/B/t/optree_concise.t b/ext/B/t/optree_concise.t
index 00a7849..22a3ff8 100644
--- a/ext/B/t/optree_concise.t
+++ b/ext/B/t/optree_concise.t
@@ -353,7 +353,7 @@ sub set_up_relative_test {
 		$h->{arg} .= ' RELATIVE' if $h->{name} eq 'leavesub';
 	    }
 	    elsif ($style eq 'scope') {
-		# supress printout entirely
+		# suppress printout entirely
 		$$format="" unless grep { $h->{name} eq $_ } @scopeops;
 	    }
 	});
diff --git a/ext/B/t/optree_varinit.t b/ext/B/t/optree_varinit.t
index f748ab8..5271a65 100644
--- a/ext/B/t/optree_varinit.t
+++ b/ext/B/t/optree_varinit.t
@@ -240,7 +240,7 @@ EONT_EONT
 checkOptree ( name	=> 'local $a=undef',
 	      prog	=> 'local $a=undef',
 	      errs      => ['Name "main::a" used only once: possible typo at -e line 1.'],
-	      note	=> 'locals are rare, probly not worth doing',
+	      note	=> 'locals are rare, probably not worth doing',
 	      bcopts	=> '-basic',
 	      strip_open_hints => 1,
 	      expect	=> <<'EOT_EOT', expect_nt => <<'EONT_EONT');
diff --git a/ext/Devel-Peek/Changes b/ext/Devel-Peek/Changes
index e87a7ca..39bbf91 100644
--- a/ext/Devel-Peek/Changes
+++ b/ext/Devel-Peek/Changes
@@ -2,7 +2,7 @@
 0.4: Hashes dumped recursively.
      Additional fields for CV added.
 0.5: Prototypes for functions supported. 
-     Strings are consostently in quotes now.
+     Strings are consistently in quotes now.
      Name changed to Devel::Peek (former ExtUtils::Peek).
 0.7:
 	New function mstat added.
diff --git a/ext/Devel-Peek/t/Peek.t b/ext/Devel-Peek/t/Peek.t
index 60fe495..ee0faa2 100644
--- a/ext/Devel-Peek/t/Peek.t
+++ b/ext/Devel-Peek/t/Peek.t
@@ -48,7 +48,7 @@ sub do_test {
 	    # things like $IVNV gave the illusion that the string passed in was
 	    # a regexp into which variables were interpolated, but this wasn't
 	    # actually true as those 'variables' actually also ate the
-	    # whitspace on the line. So it seems better to mark lines that
+	    # whitespace on the line. So it seems better to mark lines that
 	    # need to be eliminated. I considered (?# ... ) and (?{ ... }),
 	    # but whilst embedded code or comment syntax would keep it as a
 	    # legitimate regexp, it still isn't true. Seems easier and clearer
@@ -122,7 +122,7 @@ do_test('immediate constant (string)',
   CUR = 3
   LEN = \\d+');
 
-do_test('assigment of immediate constant (integer)',
+do_test('assignment of immediate constant (integer)',
         $b = 123,
 'SV = IV\\($ADDR\\) at $ADDR
   REFCNT = 1
@@ -660,7 +660,7 @@ do_test('FORMAT',
     PADNAME = $ADDR\\($ADDR\\) PAD = $ADDR\\($ADDR\\)
     OUTSIDE = $ADDR \\(MAIN\\)');
 
-do_test('blessing to a class with embeded NUL characters',
+do_test('blessing to a class with embedded NUL characters',
         (bless {}, "\0::foo::\n::baz::\t::\0"),
 'SV = $RV\\($ADDR\\) at $ADDR
   REFCNT = 1
diff --git a/ext/DynaLoader/dl_aix.xs b/ext/DynaLoader/dl_aix.xs
index eac313f..6d24428 100644
--- a/ext/DynaLoader/dl_aix.xs
+++ b/ext/DynaLoader/dl_aix.xs
@@ -8,7 +8,7 @@
  *
  *  I did change all malloc's, free's, strdup's, calloc's to use the perl
  *  equilvant.  I also removed some stuff we will not need.  Call fini()
- *  on statup...   It can probably be trimmed more.
+ *  on startup...   It can probably be trimmed more.
  */
 
 #define PERLIO_NOT_STDIO 0
diff --git a/ext/DynaLoader/dl_dlopen.xs b/ext/DynaLoader/dl_dlopen.xs
index dfa9d06..21a8ef5 100644
--- a/ext/DynaLoader/dl_dlopen.xs
+++ b/ext/DynaLoader/dl_dlopen.xs
@@ -10,7 +10,7 @@
  * 9th August 1994    - Changed to use IV
  * 10th August 1994   - Tim Bunce: Added RTLD_LAZY, switchable debugging,
  *                      basic FreeBSD support, removed ClearError
- * 29th Feburary 2000 - Alan Burlison: Added functionality to close dlopen'd
+ * 29th February 2000 - Alan Burlison: Added functionality to close dlopen'd
  *                      files when the interpreter exits
  *
  */
diff --git a/ext/DynaLoader/dl_vmesa.xs b/ext/DynaLoader/dl_vmesa.xs
index 02bb0b2..bf84322 100644
--- a/ext/DynaLoader/dl_vmesa.xs
+++ b/ext/DynaLoader/dl_vmesa.xs
@@ -2,7 +2,7 @@
  *
  * Platform:	VM/ESA, possibly others which use dllload etc.
  * Author:	Neale Ferguson (neale@mailbox.tabnsw.com.au)
- * Created:	23rd Septemer, 1998
+ * Created:	23rd September, 1998
  *
  *
  */
diff --git a/ext/File-Glob/bsd_glob.c b/ext/File-Glob/bsd_glob.c
index db36007..f891d29 100644
--- a/ext/File-Glob/bsd_glob.c
+++ b/ext/File-Glob/bsd_glob.c
@@ -815,7 +815,7 @@ glob3(Char *pathbuf, Char *pathbuf_last, Char *pathend, Char *pathend_last,
 
 
 /*
- * Extend the gl_pathv member of a glob_t structure to accomodate a new item,
+ * Extend the gl_pathv member of a glob_t structure to accommodate a new item,
  * add the new item, and update gl_pathc.
  *
  * This assumes the BSD realloc, which only copies the block when its size
diff --git a/ext/Hash-Util-FieldHash/lib/Hash/Util/FieldHash.pm b/ext/Hash-Util-FieldHash/lib/Hash/Util/FieldHash.pm
index cf65036..2414649 100644
--- a/ext/Hash-Util-FieldHash/lib/Hash/Util/FieldHash.pm
+++ b/ext/Hash-Util-FieldHash/lib/Hash/Util/FieldHash.pm
@@ -753,7 +753,7 @@ incompatibility of object bodies.
 =head1 GUTS
 
 To make C<Hash::Util::FieldHash> work, there were two changes to
-F<perl> itself.  C<PERL_MAGIC_uvar> was made avalaible for hashes,
+F<perl> itself.  C<PERL_MAGIC_uvar> was made available for hashes,
 and weak references now call uvar C<get> magic after a weakref has been
 cleared.  The first feature is used to make field hashes intercept
 their keys upon access.  The second one triggers garbage collection.
diff --git a/ext/Hash-Util-FieldHash/t/11_hashassign.t b/ext/Hash-Util-FieldHash/t/11_hashassign.t
index 8e25811..e492fa2 100644
--- a/ext/Hash-Util-FieldHash/t/11_hashassign.t
+++ b/ext/Hash-Util-FieldHash/t/11_hashassign.t
@@ -12,7 +12,7 @@ my @comma = ("key", "value");
 # The peephole optimiser already knows that it should convert the string in
 # $foo{string} into a shared hash key scalar. It might be worth making the
 # tokeniser build the LHS of => as a shared hash key scalar too.
-# And so there's the possiblility of it going wrong
+# And so there's the possibility of it going wrong
 # And going right on 8 bit but wrong on utf8 keys.
 # And really we should also try utf8 literals in {} and => in utf8.t
 
diff --git a/ext/IPC-Open3/lib/IPC/Open3.pm b/ext/IPC-Open3/lib/IPC/Open3.pm
index 658a12e..983d1a6 100644
--- a/ext/IPC-Open3/lib/IPC/Open3.pm
+++ b/ext/IPC-Open3/lib/IPC/Open3.pm
@@ -172,7 +172,7 @@ sub xclose_on_exec {
 }
 
 # I tried using a * prototype character for the filehandle but it still
-# disallows a bearword while compiling under strict subs.
+# disallows a bareword while compiling under strict subs.
 
 sub xopen {
     open $_[0], $_[1] or croak "$Me: open($_[0], $_[1]) failed: $!";
diff --git a/ext/IPC-Open3/t/IPC-Open3.t b/ext/IPC-Open3/t/IPC-Open3.t
index 9bc80da..a29b550 100644
--- a/ext/IPC-Open3/t/IPC-Open3.t
+++ b/ext/IPC-Open3/t/IPC-Open3.t
@@ -147,7 +147,7 @@ else {
 $TB->current_test($test);
 
 # RT 72016
-eval{$pid = open3 'WRITE', 'READ', 'ERROR', '/non/existant/program'; };
+eval{$pid = open3 'WRITE', 'READ', 'ERROR', '/non/existent/program'; };
 if (IPC::Open3::DO_SPAWN) {
     if ($@) {
 	cmp_ok(waitpid($pid, 0), '>', 0);
diff --git a/ext/Opcode/Opcode.xs b/ext/Opcode/Opcode.xs
index 1e5f8c2..587c299 100644
--- a/ext/Opcode/Opcode.xs
+++ b/ext/Opcode/Opcode.xs
@@ -89,7 +89,7 @@ put_op_bitspec(pTHX_ const char *optag, STRLEN len, SV *mask)
 
 /* Fetch a 'bits' entry for an opname or optag (IV/PV).
  * Note that we return the actual entry for speed.
- * Always sv_mortalcopy() if returing it to user code.
+ * Always sv_mortalcopy() if returning it to user code.
  */
 
 static SV *
diff --git a/ext/POSIX/POSIX.xs b/ext/POSIX/POSIX.xs
index 92ab1bc..239f0dd 100644
--- a/ext/POSIX/POSIX.xs
+++ b/ext/POSIX/POSIX.xs
@@ -586,7 +586,7 @@ restore_sigmask(pTHX_ SV *osset_sv)
  *
  * Note that we don't free() the malloc()ed memory unless and until we call
  * malloc() again ourselves because the CRT putenv() function simply puts its
- * pointer argument into the environ[] arrary (it doesn't make a copy of it)
+ * pointer argument into the environ[] array (it doesn't make a copy of it)
  * so this memory must otherwise be leaked.
  */
 
diff --git a/ext/POSIX/lib/POSIX.pm b/ext/POSIX/lib/POSIX.pm
index b86f2a9..bcfd1aa 100644
--- a/ext/POSIX/lib/POSIX.pm
+++ b/ext/POSIX/lib/POSIX.pm
@@ -907,7 +907,7 @@ sub load_imports {
   my %export;
   @export{map {@$_} values %EXPORT_TAGS} = ();
   # Doing the de-dup with a temporary hash has the advantage that the SVs in
-  # @EXPORT are actually shared hash key sacalars, which will save some memory.
+  # @EXPORT are actually shared hash key scalars, which will save some memory.
   push @EXPORT, keys %export;
 }
 
diff --git a/ext/POSIX/t/math.t b/ext/POSIX/t/math.t
index 2923689..e04cc30 100644
--- a/ext/POSIX/t/math.t
+++ b/ext/POSIX/t/math.t
@@ -5,7 +5,7 @@ use strict;
 use POSIX;
 use Test::More tests => 14;
 
-# These tests are mainly to make sure that these arithmatic functions
+# These tests are mainly to make sure that these arithmetic functions
 # exist and are accessible.  They are not meant to be an exhaustive
 # test for the interface.
 
diff --git a/ext/POSIX/t/time.t b/ext/POSIX/t/time.t
index dad1f6a..74d4496 100644
--- a/ext/POSIX/t/time.t
+++ b/ext/POSIX/t/time.t
@@ -42,7 +42,7 @@ if ($^O eq "hpux" && $Config{osvers} >= 11.3) {
 is(asctime(localtime(0)), ctime(0), "asctime() and ctime() at zero");
 is(asctime(localtime(12345678)), ctime(12345678), "asctime() and ctime() at 12345678");
 
-# Careful!  strftime() is locale sensative.  Let's take care of that
+# Careful!  strftime() is locale sensitive.  Let's take care of that
 my $orig_loc = setlocale(LC_TIME, "C") || die "Cannot setlocale() to C:  $!";
 my $jan_16 = 15 * 86400;
 is(ctime($jan_16), strftime("%a %b %d %H:%M:%S %Y\n", localtime($jan_16)),
diff --git a/ext/SDBM_File/sdbm/biblio b/ext/SDBM_File/sdbm/biblio
index 0be09fa..f2880dc 100644
--- a/ext/SDBM_File/sdbm/biblio
+++ b/ext/SDBM_File/sdbm/biblio
@@ -52,7 +52,7 @@
 %K torek
 
 %A Rich Wales
-%T Discusson of "dbm" data base system
+%T Discussion of "dbm" data base system
 %B USENET newsgroup unix.wizards
 %D Jan. 1984
 %K rich
diff --git a/ext/SDBM_File/sdbm/dbe.c b/ext/SDBM_File/sdbm/dbe.c
index 166e64e..f2aa290 100644
--- a/ext/SDBM_File/sdbm/dbe.c
+++ b/ext/SDBM_File/sdbm/dbe.c
@@ -286,7 +286,7 @@ main(int argc, char **argv)
 	}
 
 	if (giveusage || what == YOW || argn < 1) {
-		fprintf(stderr, "Usage: %s databse [-m r|w|rw] [-crtx] -a|-d|-f|-F|-s [key [content]]\n", argv[0]);
+		fprintf(stderr, "Usage: %s database [-m r|w|rw] [-crtx] -a|-d|-f|-F|-s [key [content]]\n", argv[0]);
 		exit(-1);
 	}
 
diff --git a/ext/SDBM_File/sdbm/sdbm.3 b/ext/SDBM_File/sdbm/sdbm.3
index fe6fe76..25afcbe 100644
--- a/ext/SDBM_File/sdbm/sdbm.3
+++ b/ext/SDBM_File/sdbm/sdbm.3
@@ -186,7 +186,7 @@ you notice clustering or other effects that may be due to the hashing
 algorithm used by this package, you can override it by supplying your
 own
 .BR sdbm_hash (\|)
-routine.  Doing so will make the database unintelligable to any other
+routine.  Doing so will make the database unintelligible to any other
 applications that do not use your specialized hash function.
 .sp
 .LP
diff --git a/ext/SDBM_File/sdbm/sdbm.c b/ext/SDBM_File/sdbm/sdbm.c
index c5f7aa8..c554e52 100644
--- a/ext/SDBM_File/sdbm/sdbm.c
+++ b/ext/SDBM_File/sdbm/sdbm.c
@@ -84,7 +84,7 @@ sdbm_open(register char *file, register int flags, register int mode)
 	if (file == NULL || !*file)
 		return errno = EINVAL, (DBM *) NULL;
 /*
- * need space for two seperate filenames
+ * need space for two separate filenames
  */
 	filelen = strlen(file);
 
@@ -306,7 +306,7 @@ makroom(register DBM *db, long int hash, int need)
 		newp = (hash & db->hmask) | (db->hmask + 1);
 
 /*
- * write delay, read avoidence/cache shuffle:
+ * write delay, read avoidance/cache shuffle:
  * select the page for incoming pair: if key is to go to the new page,
  * write out the previous one, and copy the new one over, thus making
  * it the current page. If not, simply write the new page, and we are
diff --git a/ext/Socket/Socket.pm b/ext/Socket/Socket.pm
index a34fa77..c747e4b 100644
--- a/ext/Socket/Socket.pm
+++ b/ext/Socket/Socket.pm
@@ -213,7 +213,7 @@ the name of a host, and translates that to an opaque string
 (if programming in C, struct in_addr or struct in6_addr depending on the 
 address family passed in).  The host string may be a string hostname, such
 as 'www.perl.org', or an IP address.  If using an IP address, the type of
-IP address must be consistant with the address family passed into the function.
+IP address must be consistent with the address family passed into the function.
 
 This function is not exported by default.
 
diff --git a/ext/XS-APItest/APItest.xs b/ext/XS-APItest/APItest.xs
index dda137f..0ce4d51 100644
--- a/ext/XS-APItest/APItest.xs
+++ b/ext/XS-APItest/APItest.xs
@@ -466,7 +466,7 @@ test_op_list_describe(OP *o)
     return SvPVX(res);
 }
 
-/* the real new*OP functions have a tendancy to call fold_constants, and
+/* the real new*OP functions have a tendency to call fold_constants, and
  * other such unhelpful things, so we need our own versions for testing */
 
 #define mkUNOP(t, f) THX_mkUNOP(aTHX_ (t), (f))
diff --git a/ext/XS-APItest/exception.c b/ext/XS-APItest/exception.c
index 01ff912..c0de7ec 100644
--- a/ext/XS-APItest/exception.c
+++ b/ext/XS-APItest/exception.c
@@ -10,8 +10,8 @@ static void throws_exception(int throw_e)
     croak("boo\n");
 }
 
-/* Don't give this the same name as exection() in ext/Devel/PPPort/module3.c
-   as otherwise building entirely staticly will cause a test to fail, as
+/* Don't give this the same name as execution() in ext/Devel/PPPort/module3.c
+   as otherwise building entirely statically will cause a test to fail, as
    PPPort's execption() gets used in place of this one.  */
    
 int apitest_exception(int throw_e)
diff --git a/ext/XS-APItest/t/my_cxt.t b/ext/XS-APItest/t/my_cxt.t
index 4f890bc..d2d6055 100644
--- a/ext/XS-APItest/t/my_cxt.t
+++ b/ext/XS-APItest/t/my_cxt.t
@@ -1,6 +1,6 @@
 #!perl -w
 
-# test per-interpeter static data API (MY_CXT)
+# test per-interpreter static data API (MY_CXT)
 # DAPM Dec 2005
 
 my $threads;
diff --git a/ext/mro/mro.xs b/ext/mro/mro.xs
index fb28399..76a91ae 100644
--- a/ext/mro/mro.xs
+++ b/ext/mro/mro.xs
@@ -95,7 +95,7 @@ S_mro_get_linear_isa_c3(pTHX_ HV* stash, U32 level)
 		if(items == 0 && AvFILLp(seqs) == -1) {
 		    /* Only one parent class. For this case, the C3
 		       linearisation is this class followed by the parent's
-		       inearisation, so don't bother with the expensive
+		       linearisation, so don't bother with the expensive
 		       calculation.  */
 		    SV **svp;
 		    I32 subrv_items = AvFILLp(isa_lin) + 1;
diff --git a/ext/re/t/regop.t b/ext/re/t/regop.t
index c24c32f..e3ead69 100644
--- a/ext/re/t/regop.t
+++ b/ext/re/t/regop.t
@@ -51,7 +51,7 @@ foreach my $testout ( @tests ) {
 # The format below is simple. Each line is an exact
 # string that must be found in the output.
 # Lines starting the # are comments.
-# Lines starting with --- are seperators indicating
+# Lines starting with --- are separators indicating
 # that the tests for this result set are finished.
 # If you add a test make sure you update $NUM_SECTS
 # the commented output is just for legacy/debugging purposes
-- 
1.7.3.3
```
</details>

**Comment by p5pRT** [comment]
### From @abigail

On Thu\, Jan 06\, 2011 at 11&#8203;:12&#8203;:20PM \-0800\, Peter J\. Acklam via RT wrote&#8203;:
> \# New Ticket Created by  \(Peter J\. Acklam\) 
> \# Please include the string&#8203;:  \[perl \#81882\]
> \# in the subject line of all future correspondence about this issue\. 
> \# \<URL&#8203;: http&#8203;://rt\.perl\.org/rt3/Ticket/Display\.html?id=81882 >
> 


Thanks\, applied as b7b1e41b767189c1e8835fba238082ba120976b6\.

**Comment by p5pRT** [comment]
The RT System itself - Status changed from 'new' to 'open'

**Comment by p5pRT** [comment]
@iabyn - Status changed from 'open' to 'resolved'
